docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -295,7 +295,10 @@ metrics that output a single score. These metrics take as input
 `sensitive_features` to compute the maximum difference or ratio between 
 subgroups of a sensitive variable. The predefined fairness metrics offered 
 by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
-ratio/difference. Note that because these metrics are calculated using 
+ratio/difference. Either the ratio or difference can be calculated as the max, 
```

Comment:
Sorry, I must have been more confusing than helping before. We have
- group_max
- group_min
- ratio
- difference

and for the ratio/difference it can be `between_groups` or `to_overall`, but `to_overall` results in more than 1 number.

Reply:
My apologies for the confusion on my end. I replaced the sentence cited above with "The ratio and difference can be calculated `between_groups` or `to_overall`, but `to_overall` results in more than 1 number being returned.  :class:`MetricFrame` can also calculate differences and ratios between groups using these methods. For more information on available method of computing ratios or differences, view the documentation for :meth:`MetricFrame.ratio` and :meth:`MetricFrame.difference`, respectively." Let me know if that is sufficient.